### DISCUSSION TITLE
"Magnifying glass" like zoom that follows mouse cursor

### DISCUSSION THREAD

**Discussion by regul4rj0hn** [i3/i3#4979] [Q&A] [answered]
Is there any known package/tool that would provide me with a "magnifying glass" zoom-in and zoom-out that follows my mouse cursor on the screen?

I've tried to configure the following tools with a keyboard shortcut and `floating enable` but they don't quite do what I want:
- `xzoom`: can't get the source of the zoom to be my mouse pointer, just gets started on the current window title
- `zoomx`: no man entry, no help, if there are settings to do what I want I cannot tell
- `vmg`: nope, windows vibes.

Easy to do in other DEs like Gnome, KDE and XFCE; and it seems like `compiz` supports doing this, but I'm honestly really happy with i3 + `picom` as a compositor and not looking to change either.
Scraping the web for this led to numerous "full-screen" (i.e. DPI scaling) settings, and silly Zoom meeting software screen-share related posts, but nothing useful.

Any help is appreciated!

**Comment by budRich** [comment, marked as answer]
https://github.com/tsoding/boomer

**Reply by regul4rj0hn** [reply]
Not bad at all! It's got a bit of a weird behavior when I zoom-out, but it's probably because I have 3 screens - I might be missing some settings. If anything it's MIT licensed so I can adjust it to my liking. 

Thanks a bunch, I was starting to lose hope.
